src/routes/v1/apps/handlers/deleteApp.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
         const trx = await knex.transaction()
         try {
             await deleteApp(appUuid, knex, trx)
-            trx.commit()
+            await trx.commit()
```

Comment:
If we dont wait for the commit it would delete the files even if the db-transaction fails